spring-boot-samples/spring-boot-sample-cache/src/main/resources/infinispan.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <infinispan xmlns="urn:infinispan:config:7.2">
 
-	<cache-container default-cache="default">
+	<cache-container default-cache="countries">
```

Comment:
Oh ok, thx for fixing that! That looks good. With my changes, I'd only need to update the schema version and that'd be fine.